pegasus/sites.v3/code.org/views/farsiheader.haml

```diff
@@ -0,0 +1,36 @@
+:ruby
+  cookie_key = environment_specific_cookie_name '_user_type'
+  user_type = request.cookies[cookie_key]
+  user_type = "student" if user_type == "student_y"
+
+  hamburger_options = {}
+  hamburger_options[:level] = nil
+  hamburger_options[:script_level] = nil
+  hamburger_options[:lesson] = nil
+  hamburger_options[:user_type] = user_type
+  hamburger_options[:language] = request.language
+  hamburger_options[:show_gallery] = true
+  hamburger_options[:loc_prefix] = "header_"
+  hamburger_options[:page_mode] = request.cookies['pm']
+
+  header_contents_options = {}
+  header_contents_options[:user_type] = user_type
+  header_contents_options[:language] = request.language
+  header_contents_options[:loc_prefix] = "header_"
+  header_contents_options[:page_mode] = request.cookies['pm']
+
+  require 'cdo/hamburger'
+
+=inline_css 'hamburger.css'
+
+%nav.main
+  .left
```

Comment:
Is this and the `.right` div on line 36 added intentionally to center the header nav?